SimGeneral/Debugging/python/caloParticleDebugger_cfg.py

```diff
@@ -8,7 +8,7 @@
 process = cms.Process("Demo")
 
 process.load("FWCore.MessageService.MessageLogger_cfi")
-process.load('Configuration.Geometry.GeometryExtended2026D49Reco_cff')
+process.load('Configuration.Geometry.GeometryExtended2026D96Reco_cff')
```

Comment:
Do you have a specific reason to use D96, as it is not a baseline of Phase-2? If there is no need in specific Phase-2 D-whatever, please use `GeometryExtended2026Default_cff.py`. It is a placeholder where default is defined.

Reply:
I tried updating it, but for some reason this leads to merge conflicts